### PR TITLE
Doc fix and other small fixes

### PR DESCRIPTION
**Pull request by jurejesensek**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ project(sycl-gtx-solution)
 # Requirements
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
```
Comment (ProGTX) [inline CMakeLists.txt]: Why is this needed?

**Comment by ProGTX** [comment]
Sorry for the very late response, thank you for this. We may also want to add `@brief` if we're going with Doxygen.

File: CMakeLists.txt (hunk 1 above)
Comment (jurejesensek) [reply CMakeLists.txt]: This option forces ISO C++11 standard, without additional extensions (such as those in GNU's C++11 standard).
